pom.xml

```diff
@@ -271,6 +271,11 @@
             <version>1.2.5</version>
             <scope>test</scope>
         </dependency>
+<dependency>
```

Comment:
This seems to be under indented compared to the other dependencies.

Did you check that this plugin is fossed?

Reply:
Well, I need to get is checked for FOSS...